### ISSUE TITLE
rfc2136 support

### ISSUE THREAD

**Issue by Slach** [vagrant-landrush/landrush#355]
i try deploy local kubernetes cluster inside vagrant
and try use external-dns

I'm very interested add support rfc2136 to landrush DNS server in "unsecured" mode, for example
https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/rfc2136.md

could somebody suggest me code places where i can see and try implements rfc2136 support in landrush?

**Comment by davividal** [comment]
I don't think that landrush would be the right place to start, as it is a "frontend" to https://github.com/socketry/rubydns .

While you are at that, what do you think about helping with https://github.com/vagrant-landrush/landrush/issues/312 ? I wanted to give this issue a try for a while now, but I didn't get time for that so far.
